docs/source/dev_guide/glossary.rst

```diff
@@ -13,6 +13,24 @@ will be used in the following of this guide.
 
 .. contents::
 
+Set up ecpy in developper mode
+------------------------------
+Here we describe a simple workflow for developpers to contribute to ecpy. 
+It is convenient to use a git GUI such as smartgit. Then, create a repository 
+on your computer where you can clone all the ecpy
+repositories (ecpy, ecpy_hqc_legacy, ecpy_pulses, etc). On your command 
+terminal, navigate to the folder
+of each repository, and type the following command:
+    $ python setup.py develop
```

Comment:
It may be worth detailing the fact that this allows ecpy components to be well detected while directly taking account any change made to the code.

Reply:
ok